packages/frontend/src/components/common/PasswordValidationMessages.tsx

```diff
@@ -0,0 +1,29 @@
+import { PasswordValidationResult } from '@lightdash/common';
+import { Stack, Text } from '@mantine/core';
+import React from 'react';
+
+const PasswordValidationMessages: React.FC<PasswordValidationResult> = ({
+    isLengthValid,
+    hasLetter,
+    hasNumberOrSymbol,
+}) => {
+    const getColor = (status: boolean) => {
+        return status ? 'green.6' : 'red.6';
+    };
+
+    return (
+        <Stack spacing="xxs">
+            <Text color={getColor(isLengthValid)}>
+                Must be at least 8 characters long
+            </Text>
+            <Text color={getColor(hasNumberOrSymbol)}>
+                Must contain at least one symbol or number
+            </Text>
+            <Text color={getColor(hasLetter)}>
+                Must contain at least one letter
+            </Text>
+        </Stack>
+    );
+};
```

Comment:
would be nice to show validation errors on actual input rather than on a separate view

here are a lot of examples including how to integrate with zod

https://mantine.dev/form/validation/